packages/grpcts/src/Client.ts

```diff
@@ -9,14 +10,20 @@ export interface ClientTrace {
 
 type GrpcClient = grpc.Client & { [implementation: string]: any };
 
+export interface Logger {
+  info(message: string, payload?: any): void;
+}
+
 export class Client {
   public client: GrpcClient;
   constructor(
-    definition: grpc.ServiceDefinition<any>,
+    readonly definition: grpc.ServiceDefinition<any>,
     address: string,
     credentials: grpc.ChannelCredentials = grpc.credentials.createInsecure(),
     public readonly trace?: ClientTrace,
-    options?: object
+    options?: object,
+    private readonly logger?: Logger,
```

Comment:
The constructor function signature grew a lot - should we replace it with a single object?

Reply:
i don't really want to deep and refactor it right now. want to check it this actually works or not.